docs/source/chainermn/model_parallel/model_parallel_on_chainermn.rst

```diff
@@ -0,0 +1,202 @@
+Model Parallel on ChainerMN
+===========================
+
+
+Step 1: Communicators
+~~~~~~~~~~~~~~~~~~~~~
+
+To perform multi-node communications, a *communicator* is needed.
+Basic usages are the same with the case of the data parallel, see :doc:`../tutorial/step1_communicators_optimizers`::
+
+    comm = chainermn.create_communicator()
+
+If you want to define collective communications among limited number of processes later, it is useful to split the communicator::
+
+    subcomm = comm.split(comm.rank % 2, comm.rank)
+
+.. figure:: ../../../image/model_parallel/comm_split.png
+    :align: center
+    :scale: 50%
+
+For further detail about the communicator split, please refer to `MPI tutorial <http://mpitutorial.com/tutorials/introduction-to-groups-and-communicators/>`__.
+
+
+
+Step 2: Datasets and Iterators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In model parallel, there are three types of processes regarding to datasets.
```

Comment:
"In model parallel training, all processes belong to at least one of following dataset input patterns."